src/service/vaults.ts

```diff
@@ -418,14 +401,16 @@ export const makeAdjustVaultOffer = async ({
   collateral,
   debt,
 }: AdjustParams) => {
-  const invitationSpec = {
-    source: 'continuing',
-    previousOffer: vaultOfferId,
-    invitationMakerName: 'AdjustBalances',
-  };
-
   const { importContext, offerSigner } = appStore.getState();
 
+  const invitationSpec = importContext.fromBoard.serialize(
```

Comment:
why does this need to be serialized now?

Reply:
The wallet app now expects the invitation spec to be capdata. This supports the `agoricContract` proposal which has a `Brand` as an arg. It was easier this way than making the wallet-app check if it's capdata or not.

Reply:
ok, makes sense that what goes between the apps is marshaled.